src/components/DataArchives/DataArchives.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: 'white'
+  },
+  main: {
+    paddingBottom: '3rem',
+    [theme.breakpoints.up('xs')]: {
```

Comment:
On small screen, you usually want content to go from left edge to right edge of the phone i.e. no left/right padding/margins.